### PR TITLE
Push auth

### PR DESCRIPTION
**Pull request by mdellweg**
This is bases on #103

### CODE REVIEW

File: pulp_container/app/registry.py [hunk 1]
```diff
@@ -242,7 +242,6 @@ async def get_by_digest(self, request):
         distribution = self._match_distribution(path)
         self._permit(request, distribution)
         repository_version = distribution.get_repository_version()
-        log.info(digest)
```
Comment (dkliban) [inline pulp_container/app/registry.py:245]: Good. We were logging a lot :)

File: pulp_container/app/token_verification.py [hunk 2]
```diff
@@ -24,17 +25,14 @@ def _decode_token(encoded_token):
...
-            decoded_token = {"access": []}
+        decoded_token = jwt.decode(encoded_token, public_key.read(), **JWT_DECODER_CONFIG)
     return decoded_token
 
 
 def _access_scope(request):
     repository_path = request.resolver_match.kwargs.get("path", "")
 
-    if request.method in ["GET", "HEAD", "OPTIONS"]:
+    if request.method in SAFE_METHODS:
```
Comment (dkliban) [inline pulp_container/app/token_verification.py:35]: Nice use of predefined constant from DRF.

File: pulp_container/app/registry.py (hunk 1 above)
Comment (mdellweg) [reply pulp_container/app/registry.py:245]: This really looks like forgotten debugging code (no accompanying explanation around the bare digest).

File: pulp_container/app/token_verification.py (hunk 2 above)
Comment (mdellweg) [reply pulp_container/app/token_verification.py:35]: Yeah, i am still constantly learning about DRF. ;)

**Pull request merged**